### PR TITLE
Inverter support

### PR DESCRIPTION
**Pull request by fboundy**
More fixes to get the Solis working

### CODE REVIEW

**Comment by fboundy** [comment]
Don't merge this yet - I think it's breaking something!

**Review by springfall2008 — APPROVED** [review]

**Comment by springfall2008** [comment]
Okay, I've approved it for once you are ready

**Comment by fboundy** [comment]
Might me fine - need to check everything kicks in OK later today when charging starts again. Loads of sun today so will have to wait and see!

**Pull request merged**
